include/drivers/pwm.h

```diff
@@ -43,6 +76,56 @@ typedef int (*pwm_pin_set_t)(const struct device *dev, uint32_t pwm,
 			     uint32_t period_cycles, uint32_t pulse_cycles,
 			     pwm_flags_t flags);
 
+/**
+ * @typedef pwm_capture_callback_handler_t
+ * @brief PWM capture callback handler function signature
+ *
+ * Note: The callback handler will be called in interrupt context.
+ *
+ * @param dev Pointer to the device structure for the driver instance.
+ * @param flags PWM capture flags
+ * @param period Pointer to the memory to store the captured PWM period width
+ *               (in clock cycles). HW specific.
+ * @param pulse Pointer to the memory to store the captured PWM pulse width (in
+ *              clock cycles). HW specific.
```

Comment:
These entries need to be corrected. The callback function signature specifies `pwm` parameter, not `flags`, and the other two parameters are named `period_cycles` and `pulse_cycles`, respectively.

Reply:
Good catch. Fixed, thanks.